### PR TITLE
[No QA] Update botify workflows to use new signing key

### PR DESCRIPTION
**Pull request by Jag96**
### Details
https://github.com/Expensify/App/pull/6371 fixed the issue with the createNewVersion workflow, but the [cherry-pick workflow](https://github.com/Expensify/App/runs/4269063989?check_suite_focus=true) failed. This PR updates all botify flows that push to git to use the updated signing key.

A future improvement here would be to create a separate workflow to handle setting up the git config for botify to reduce duplicate code, so once CPs are fixed we can create a follow-up issue to do that.

### Fixed Issues
Fixes [broken GH action](https://github.com/Expensify/App/runs/4269063989?check_suite_focus=true)

### Tests
Merge and CP this, confirm the workflow is successful and that the commits are signed.

### QA Steps
None

### Tested On

- [ ] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### CODE REVIEW

File: .github/workflows/cherryPick.yml [hunk 1]
```diff
@@ -69,6 +69,23 @@ jobs:
...
+      - name: Import Botify GPG Key
+        if: github.actor == 'OSBotify'
+        run: cd .github/workflows && gpg --import OSBotify-private-key.asc
+
+      - name: Set up git for Botify
+        if: github.actor == 'OSBotify'
+        run: |
+          git config user.signingkey DBF63700F60F5530
+          git config commit.gpgsign true
+          git config user.email infra+osbotify@expensify.com
```
Comment (AndrewGable) [inline .github/workflows/cherryPick.yml:87]:
```suggestion
          git config user.name OSBotify
          git config user.email infra+osbotify@expensify.com
```

File: .github/workflows/cherryPick.yml [hunk 2]
```diff
@@ -69,6 +69,23 @@ jobs:
...
+          LARGE_SECRET_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
+
+      - name: Import Botify GPG Key
+        if: github.actor == 'OSBotify'
+        run: cd .github/workflows && gpg --import OSBotify-private-key.asc
+
+      - name: Set up git for Botify
+        if: github.actor == 'OSBotify'
+        run: |
+          git config user.signingkey DBF63700F60F5530
```
Comment (nickmurray47) [inline .github/workflows/cherryPick.yml:85]: Does this `signingkey` need to be stored as an environment variable?

Comment (AndrewGable) [reply .github/workflows/cherryPick.yml:85]: Nope, it's a public key ID and exposed by the GitHub UI when you click on `Verified`

Screen Shot 2021-11-19 at 2 58 40 PM

File: .github/workflows/cherryPick.yml (hunk 1 above)
Comment (Jag96) [reply .github/workflows/cherryPick.yml:87]: `user.name` is set below but we can set it here to be explicit

**Review by AndrewGable — APPROVED** [review]

**Comment by AndrewGable** [comment]
Skipping E2E tests to unblock CPs.

**Comment by botify** [comment]
@AndrewGable looks like this was merged without passing tests. Please add a note explaining why this was done and remove the `Emergency` label if this is not an emergency.

**Comment by AndrewGable** [comment]
@botify https://github.com/Expensify/App/pull/6374#issuecomment-974526574

**Pull request merged**